### PR TITLE
Named enrolment urls

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-448)
[this task](https://uktrade.atlassian.net/browse/ED-450)

urls are now:

/register/company-number
/register/company-name
/register/exports
/register/email
/register/mobile
/register/mobile-verify

### CODE REVIEW

File: enrolment/tests/test_views.py [hunk 1]
```diff
@@ -17,10 +17,22 @@
...
 )
-from enrolment import forms, helpers, validators
 from sso.utils import SSOUser
 
 
+valid_user_data_step = {
+    'enrolment_view-current_step': EnrolmentView.StepNames.user,
+    EnrolmentView.StepNames.user + '-mobile_number': '07507405138',
+    EnrolmentView.StepNames.user + '-mobile_confirmed': '07507405138',
+    EnrolmentView.StepNames.user + '-terms_agreed': True,
```
Comment (richtier) [inline enrolment/tests/test_views.py]: this might look a mess - but it avoids 'magic strings'.

File: enrolment/views.py [hunk 2]
```diff
@@ -55,25 +55,33 @@ def form_valid(self, form):
         return TemplateResponse(self.request, self.success_template)
 
 
-class EnrolmentView(SSOLoginRequiredMixin, SessionWizardView):
+class EnrolmentView(SSOLoginRequiredMixin, NamedUrlSessionWizardView):
+
+    class StepNames:
```
Comment (konradko) [inline enrolment/views.py]: why not just use class attributes?

Comment (konradko) [inline enrolment/views.py]: I think you just like javascript's dot notation :-P Better to use dict if you need a data structure,  although constants / class attributes still feel most clean / DRY to me here.

Comment (richtier) [inline enrolment/views.py]: I will make them constants on the view. nice and DRY and self-contained and flat and what not.

**Pull request merged**